src/main/java/bio/terra/app/controller/DataRepositoryServiceApiController.java

```diff
@@ -104,10 +104,12 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     @Override
     public ResponseEntity<DRSObject> getObject(
         @PathVariable("object_id") String objectId,
-        @RequestParam(value = "expand", required = false, defaultValue = "false") Boolean expand) {
+        @RequestParam(value = "expand", required = false, defaultValue = "false") Boolean expand
```

Comment:
It's not a huge deal but IMO it would be better if the changes in this file were completely reverted back to what's on `develop`, so it's clear that no changes to this code were necessary.